plugin/storage/integration/kafka_test.go

```diff
@@ -91,6 +93,10 @@ func (s *KafkaIntegrationTestSuite) initialize(t *testing.T) {
 	}
 	options.InitFromViper(v)
 	traceStore := memory.NewStore()
+	_, err = builder.CreateConsumer(logger, metrics.NullFactory, traceStore, options)
+	require.Error(t, err, "invalid version `0`")
+	command.ParseFlags([]string{"--kafka.consumer.protocol-version", "2.0.0"})
```

Comment:
add new line to separate logically different group
```suggestion

	command.ParseFlags([]string{"--kafka.consumer.protocol-version", "2.0.0"})
```